sonic-net/sonic-buildimage
# [Ufispace] Update the transceiver control virtual address to avoid an address conflict with SPD

#### Why I did it

Update the transceiver control virtual address to avoid an address conflict with SPD
* s6301-56st
* s7801-54xs
* s8901-54xc
* s9110-32x


##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it

Update the PDDF config files.

#### How to verify it



Run the PDDF commands and show commands to check the sysfs and transceiver status.

[s6301-56st.txt](https://github.com/user-attachments/files/16450946/s6301-56st.txt)
[s7801-54xs.txt](https://github.com/user-attachments/files/16450947/s7801-54xs.txt)
[s8901-54xc.txt](https://github.com/user-attachments/files/16450944/s8901-54xc.txt)
[s9110-32x.txt](https://github.com/user-attachments/files/16450945/s9110-32x.txt)



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
